invenio_app_rdm/records_ui/views/decorators.py

```diff
@@ -365,20 +367,169 @@ def view(**kwargs):
     return view
 
 
-def add_signposting(f):
-    """Add signposting link to view's response headers."""
+def _get_header(rel, value, link_type=None):
+    header = f'<{value}> ; rel="{rel}"'
+    if link_type:
+        header += f' ; type="{link_type}"'
+    return header
+
+
+def _get_signposting_cite_as(record):
+    """Release self url points to RDM record.
+
+    It points to DataCite URL if the integration is enabled, otherwise it points to the HTML URL.
+    """
+    doi_url = record["links"].get("doi")
+    html_url = record["links"]["self_html"]
+    return _get_header("cite-as", doi_url or html_url)
+
+
+def _get_signposting_types(record):
+    resource_type = record["metadata"]["resource_type"]
+    props = get_vocabulary_props(
+        "resourcetypes",
+        [
+            "props.schema.org",
+        ],
+        resource_type["id"],
+    )
+    url_schema_org = props.get("schema.org")
+    return [
+        _get_header("type", url_schema_org),
+        _get_header("type", "https://schema.org/AboutPage"),
+    ]
+
+
+def _get_signposting_authors(record):
+    authors = []
+    # Limit authors to the first 10.
+    for creator in islice(record["metadata"]["creators"], 0, 10):
+        for identifier in creator["person_or_org"].get("identifiers", []):
+            if identifier["scheme"] == "orcid":
+                authors.append(
+                    _get_header(
+                        "author", "https://orcid.org/" + identifier["identifier"]
+                    )
+                )
```

Comment:
Should we support other schemes like ROR, etc?

Reply:
It might be that the safer option would be to use something like `idutils.to_url(identifier, scheme)` which will consistently produce a link.

Reply:
I'm now relying on [invenio_rdm_records/resources/serializers/signposting/schema.py's serialize_author](https://github.com/inveniosoftware/invenio-rdm-records/blob/938b7cf588d986c8e15de10d5d81baae17e2fddc/invenio_rdm_records/resources/serializers/signposting/schema.py#L35) which picks the first linkable ID.